packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -9,9 +9,50 @@ import { getUUIDFrom } from "../schemas/uuid";
 import { asyncHandler, getETag, getFromParamsOrFail, getFromQueryOrFail } from "../util";
 import { PatientLinksDTO, dtoFromCW } from "./dtos/linkDTO";
 import { linkCreateSchema } from "./schemas/link";
+import { getFacilities } from "../../command/medical/facility/get-facility";
+import { getPatients } from "../../command/medical/patient/get-patient";
+import { PatientUpdateCmd, updatePatient } from "../../command/medical/patient/update-patient";
 
 const router = Router();
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/patient/update-all
+ *
+ * Triggers an update for all of a cx's patients without changing any
+ * demographics. The point of this is to trigger an outbound XCPD from
+ * CommonWell to Carequality so new patient links are formed.
+ *
+ *
+ * @param req.query.cxId The customer ID.
+ * @return 204 No Content
```

Comment:
nit: sync with the code (update this to 200 or the code to 204)

Reply:
🕵️ done